packages/auth/src/authSession.ts

```diff
@@ -47,6 +52,8 @@ export type CreateAuthSessionParams = {
   authType: AuthType;
   /** Credential ID for passkey authentication */
   credentialId?: string;
+  /** Expiration timestamp - if not provided, defaults to 15 minutes from now */
+  expirationDateMs?: number;
```

Comment:
The comment and variable name are a bit imprecise: they make it sound like you're providing the timestamp at which the session expires, rather than the length of the session. I'd consider naming the variable either just `expirationMs` or maybe something like `sessionDurationMs`, and adjusting the comment.

(I realize the variable name already existed in the code, but I didn't notice it before and now it's becoming part of the public API so needs more attention.)

Reply:
+1